pynest/examples/clopath_synapse_small_network.py

```diff
@@ -116,7 +116,7 @@
 
```

Comment:
Both Clopath examples: Remove unnecessary `1`s from `Create()` calls.

Reply:
This is something that may need some more thought. Some ", 1"s are instructive, some are not. While I completely agree that `Create("weight_recorder")` is fine without the 1, something like `Create("poisson_generator")` or `Create("parrot_neuron")` is very different. The former will be the same most of the time and does usually not scale with the experiment, while the latter may only be a single one because there is exactly one sequence generated, not one per neuron etc.
The transition from `Create("parrot_neuron")` to `Create("parrot_neuron", ng)` is much bigger than from `Create("parrot_neuron", 1)`. On the other hand, for the recorders it may be a bad idea to hint the user to uselessly scaling the number with the neurons.
Looking didactically into each example and deciding if ",1)" point in the right direction is imho however out of the scope of this PR. I fixed some where it felt more natural, but not all and not with a consistent scheme in mind. I'll ask around for opinions, but this PR should probably better proceed with the ", 1"s as they are. We can always do another PR once we know what to do best here.